pkg/tkctl/executor/executor.go

```diff
@@ -30,7 +31,7 @@ import (
 	"k8s.io/kubernetes/pkg/kubectl"
 	"k8s.io/kubernetes/pkg/kubectl/cmd"
 	"k8s.io/kubernetes/pkg/util/interrupt"
-	"time"
+	v1 "k8s.io/kubernetes/staging/src/k8s.io/api/core/v1"
```

Comment:
why use it instead of `"k8s.io/api/core/v1"`?

Reply:
oh, this is a mistake. I'll fix it.

Reply:
fixed